### PR TITLE
Add Celsius-sign to swimming water temperatures

### PR DESCRIPTION
**Pull request by mhieta**
Pulkka - Celsius sign

Refs HUL-19

### CODE REVIEW

**Comment by terovirtanen** [comment]
**SPORTS-MAINTENANCE-DASHBOARD branch is deployed to platta: https://pulkka-ulkoliikunta-pr33.dev.hel.ninja** :rocket::rocket::rocket:

File: src/components/UnitStatusSummary.js [hunk 1]
```diff
@@ -13,6 +13,15 @@ const SHORT_DESCRIPTIONS = {
   live_swimming_water_temperature: 'Automaattinen lämpötilamittaus'
 };
 
+function getUnitObservationText(observationName, observationProperty) {
+  console.log(observationProperty);
```
Comment (japauliina) [inline src/components/UnitStatusSummary.js:17]: Is this printing needed?

Comment (mhieta) [reply src/components/UnitStatusSummary.js]: Not needed, removed it.

**Comment by terovirtanen** [comment]
**SPORTS-MAINTENANCE-DASHBOARD branch is deployed to platta: https://pulkka-ulkoliikunta-pr33.dev.hel.ninja** :rocket::rocket::rocket:

**Review by japauliina — APPROVED** [review]

**Pull request merged**